jobs/build/scan-osh/Jenkinsfile

```diff
@@ -49,75 +43,63 @@ node('ocp-artifacts') {
         commonlib.checkMock()
 
         stage("initialize") {
-                if (!(params.BUILD_NVRS || params.RPM_NVRS)) {
-                    error("Either BUILD_NVRS or RPM_NVRS is required")
-                }
-        }
+            if (!params.NVRS) {
+                error("NVRS field should not be empty")
+            }
 
-        stage("images-scan") {
             buildlib.cleanWorkdir("./artcd_working")
             sh "mkdir -p ./artcd_working"
+        }
 
-            if (params.BUILD_NVRS) {
-                def builds = params.BUILD_NVRS.split(',')
-
-                for (String build : builds) {
-                    def cmd = [
+        stage("kick-off-scans") {
+            def nvrs = params.NVRS.split(',')
+            def cmd = []
+            errors = []
+            for (String nvr : nvrs) {
+                if (nvr.contains("container")) {
```

Comment:
We can be stricter here and check for ends with `-container` 
Just a suggestion though, this also is fine

Reply:
ahh right, but since this is Groovy, wanted to make it as simple as possible, hope this one is fine for now?